### DISCUSSION TITLE
Error when using symbol currency

### DISCUSSION THREAD

**Discussion by ngothoai** [mpdf/mpdf#1841] [Q&A]
### Guidelines

- [X] I understand that [if I fail to provide all required details, this issue may be closed without review](https://github.com/mpdf/mpdf/blob/development/.github/CONTRIBUTING.md).

### Description of the bug

Hello,
When i used to the library mpdf. I found that: When I using symbol in here: https://www.w3schools.com/charsets/ref_utf_currency.asp
Then I exported it is not show.
Can you check this issue?
Regard.

### mPDF version

8.0

### PHP Version and environment (server type, cli provider etc., enclosing libraries and their respective versions)

7.4

### Reproducible PHP+CSS+HTML snippet suffering by the error

Hello,
When i used to the library mpdf. I found that: When I using symbol in here: https://www.w3schools.com/charsets/ref_utf_currency.asp
Then I exported it is not show.
Can you check this issue?
Regard.

**Comment by ssieb** [comment]
You didn't include a reproducible example.

**Comment by ngothoai** [comment]
Thanks for your reply.
Example with my code: 
https://prnt.sc/Cc0iNTSPmfDu

Result error:
https://prnt.sc/tJeNjRD_EPI9

My symbol currency:  ₼

**Comment by ssieb** [comment]
You still haven't provided an example, only a screenshot of part of some incomplete html.
Make sure your font actually supports that character.  It looks like it doesn't.

**Comment by ngothoai** [comment]
I read your document, But in your library is not support this charector special : ₼
https://mpdf.github.io/reference/codepages-glyphs/win-1252.html

**Comment by ngothoai** [comment]
```
<tr>
                    <td class="title"><?php echo esc_html__("Subtotal: ",'traveler-export') ?></td>
                    <td class="value">₼ 150.00</td>
                </tr>
```
This is my HTML code for export PDF

**Comment by finwe** [comment]
You need to use a font that includes the currency character, Default DejaVu as far as I can see does not. Not a bug of the library.

**Comment by ngothoai** [comment]
I am using default DejaVu font

**Comment by finwe** [comment]
And it obviously does not include required character.

**Comment by ngothoai** [comment]
I am try other  charactor special but only with ₼ then this issue is happen
